finish/README.txt

```diff
@@ -25,50 +23,42 @@ BankService.java file also contains code that automatically fails the first 2
 requests that are made to the checkBalance service so as to demonstrate the states 
 of a circuit breaker. 
 
-The @Fallback annotation and the fallback method are in a separate file, 
-BankServiceWithFallback.java. The fallback method is invoked whenever a request 
-to the service fails or when the circuit is open. When the fallback is running 
-the method immediately returns the following message: 
-	"The last known balance of your account is $10,293". 
+The @Fallback annotation and the fallback method are demonstrated in the same source 
+file. The fallback method is invoked whenever a request to the service fails or when 
+the circuit is open. When the circuit is opened, the request is not allowed to go 
+through and the fallback is called immediately. 
 
-To access the sample application with the checkBalance service, visit the 
-following URL from your browser: 
+To access the sample application with the checkBalance service that includes
+a fallback method, visit the following URL from your browser: 
       http://localhost:9080/circuitBreakerSample/checkBalance 
 The checkBalance service simulates access failures in order to demostrate the 
 circuit breaker in action. The first 2 attempts to access the service will fail. 
-Therefore, after the initial invocation of the checkBalance service the output 
+The fallback is called when the first request fails. The output from the fallback 
 will show the following message:
-      "The system is down. Try again later."       
+      "The last known balance of your account is $10,293". 
+
 Refresh the browser. This second invocation fails in the same manner. The 
 circuit has now reached its failure threshold and enters into an open state. 
 It remains in an open state for 5 seconds before switching to a half-open state. 
 If a request to the checkBalance service occurs while the circuit is in an open 
-state the output immediately displays a slightly different response that displays 
-the following message:
-      "The system is still down. Try again later." 
-However, if you wait 5 seconds before refreshing the browser, then the circuit 
-will be in a half-open state and the request is allowed to process. The code 
-in BankService.java only simulates a failure for the first two requests to the 
-checkBalance service, so this request will be successful and shows the  
-following message: 
-	"Your account current balance is $10,293". 
+state, the output immediately displays the same message as requests are not allowed
```

Comment:
'state, the output immediately displays the same message **from the fallback method because** requests are not allowed to go through **to** the checkBalance service.

Reply:
Thank you. fixed.